src/SchemaUtils.elm

```diff
@@ -534,6 +556,21 @@ typeToEncoder qualify namespace type_ =
         Common.Bool ->
             CliMonad.succeed Gen.Json.Encode.call_.bool
 
+        Common.Enum constructors ->
+            CliMonad.succeed
+                (\expr ->
+                    Elm.Case.string
+                        expr
+                        { cases =
+                            List.map
+                                (\constructor ->
+                                    ( constructor, Gen.Json.Encode.values_.string )
+                                )
+                                constructors
+                        , otherwise = Gen.Json.Encode.call_.string expr
+                        }
+                )
```

Comment:
Isn't this just a convoluted way to call `Json.Encode.string` in all cases anyway? :thinking:

Reply:
Actually, this generates wrong code :thinking:
```elm
        , ( "type"
          , case rec.type_ of
                "ValuePropositionSummary" ->
                    Json.Encode.string

                _ ->
                    Json.Encode.string rec.type_
          )
```

Reply:
What if we just
```elm
-- Elm.Case.string
--     expr
--     { cases =
--         List.map
--             (\constructor ->
--                 ( constructor, Gen.Json.Encode.call_.string expr )
--             )
--             constructors
--     , otherwise = Gen.Json.Encode.call_.string expr
--     }
Gen.Json.Encode.call_.string expr
```
The commented code left there for when we switch to proper typed enums

Reply:
I'll just remove the commented out part. If I _really_ need to reference it I can search git logs